src/main/java/frc/robot/commands/ShooterTuner.java

```diff
@@ -34,14 +34,14 @@ public ShooterTuner(Shooter shooter, Carousel carousel, CarouselCommand cc) {
   @Override
   public void initialize() {
     shooter.calibratePID(0.000145, 1e-8, 0, 6.6774 * 0.00001);
-    shooter.setLEDRing(true);
+    shooter.vision.setLEDRing(true);
```

Comment:
This might be the reason we exposed setLEDRing, but are you sure we can't use shooter.vision.setMode here?